sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -52,6 +52,8 @@ @implementation OOSkinViewController
 static NSString *kFrameChangeContext = @"frameChanged";
 static NSString *kViewChangeKey = @"frame";
 
+NSString *const OOOoyalaPlayerFullScreenChangedNotification = @"fullScreenChanged";
```

Comment:
@wunnath Yes these are specific to SkinSDK and so they are here :)
@lental I will change name :)